next.config.js

```diff
@@ -1,4 +1,8 @@
 module.exports = {
+    images: {
+        loader: 'imgix',
+        path: ''
+    },
```

Comment:
Image optimization with the `next/Image` component is not enabled for exported applications because optimization is done on demand, so we cannot use the default configuration.

If we are going to do an export and we want to use `next/Image` we must use an Optimization cloud provider different from default, nextJS gives a list [here](https://nextjs.org/docs/basic-features/image-optimization#loader)